feat: Implement linting feature and pre-commit mode

svdimchenko opened the pull request:
Implements https://github.com/marcosschroh/dc-avro/issues/68

marcosschroh commented:
Thanks for the PR @svdimchenko 

Could you [document](https://marcosschroh.github.io/dc-avro/commands/) this new command :-) ? Also, we should explain the difference between `validate-schema` and `lint`.

svdimchenko commented:
> Thanks for the PR @svdimchenko
> 
> Could you [document](https://marcosschroh.github.io/dc-avro/commands/) this new command :-) ? Also, we should explain the difference between `validate-schema` and `lint` and the `pre-commit` hook

@marcosschroh I've added corresponding docs and fixed formatting. Will you release `0.7.0` version ?
